Cargo.toml

```diff
@@ -26,6 +26,12 @@ inspect = ["bevy-inspector-egui"]
 serialize = ["serde"]
 
 [dependencies]
-bevy = { version = "0.11" }
+bevy = { version = "0.11", default-features = false, features = [
+    "bevy_render",
+    "bevy_ui"
+] }
 bevy-inspector-egui = { version = "0.19", optional = true }
 serde = { version = "^1", features = ["derive"], optional = true }
+
+[dev-dependencies]
+bevy = "0.11"
```

Comment:
Because in the development dependencies it comes complete? I think it would be better to remove these lines, mostly because what you did above definitely helps to reduce compile times in development and lightens the times

Reply:
You still need a few more features to have a functional bevy application (for the examples). I can't remember exactly which and it changes for each bevy release, so the simplest is to just use the default, to ease maintenance burden.

Reply:
Right, I had forgotten that the examples need certain features, let me test how much it affects this change by implementing the library in a project and based on that I will let you know if we proceed or see any alternative

Reply:
We used default features = false in matchbox. And it broke in 0.11.0 because the webgl2 feature was introduced, and we accidentally got webgpu instead, then again in 0.11.3 because we were missing tonemapping or ktx2 or something.

Reply:
FYI if you want the minimal dependencies route, it's:

bevy = { version = "0.11", default-features = false, features = [
    "bevy_winit",
    "png",
] }

also add webgl2 unless you want experimental webgpu rendering on web.

I ended up doing it this way, because I don't want to install alsa on gh runners for cargo test to work in ci.